devices/debian.py

```diff
@@ -402,6 +402,41 @@ def start_sshd_server(self):
         self.sendline('/etc/init.d/ssh reload')
         self.expect(self.prompt)
 
+    def get_tftp_cfg_dest_path(self, board=None):
+        return '/tftpboot' # can this change?
```

Comment:
Not so far, but we can just make this an arg in the function below with the default being /tftpboot for now?

Reply:
ACK